onadata/apps/api/viewsets/export_viewset.py

```diff
@@ -38,7 +40,7 @@ class ExportViewSet(DestroyModelMixin, ReadOnlyModelViewSet):
     permission_classes = [ExportDjangoObjectPermission]
 
     def retrieve(self, request, *args, **kwargs):
-        export = self.get_object()
+        export = get_object_or_404(self.queryset, pk=kwargs.get('pk'))
```

Comment:
Why? I would think `self.get_object()` does exactly this, I will say this is straying from the way Django rest framework does it.